gui/mozregui/ui/inbound.ui

```diff
@@ -56,6 +56,20 @@
        </property>
       </widget>
      </item>
+     <item row="2" column="0">
+      <widget class="QLabel" name="label_4">
+       <property name="text">
+        <string>Build Type</string>
+       </property>
+      </widget>
+     </item>
+     <item row="2" column="1">
+      <widget class="QComboBox" name="build_type">
+       <property name="toolTip">
+        <string>Choose a build type you want, notice only opt is available most of time</string>
```

Comment:
Please remove "notice only opt is available most of time". (we handle every case for inbound, the warning was for users using command lines and nightlies).